src/player.cpp

```diff
@@ -4835,3 +4961,39 @@ void Player::updateRegeneration()
 		condition->setParam(CONDITION_PARAM_MANATICKS, vocation->getManaGainTicks() * 1000);
 	}
 }
+
+bool Player::addLoyalty(int32_t points)
+{
+	if (loyaltyPoints == 0) {
```

Comment:
I might not understand how this work, but why would you not allow adding loyalty points if loyaltyPoints is 0?

Reply:
Not sure, but I think it doesnt matter, so just let it work even with loyalty system disabled